Dockerfile

```diff
@@ -19,6 +19,11 @@ COPY secrets /opt/ml-api-adapter/secrets
 
 FROM node:12.16.0-alpine
 
+WORKDIR /opt/central-ledger/cert
```

Comment:
The directory seems to be incorrect here?

Reply:
Copy Paste issue... will be removed due to below comment: